### PR TITLE
Update team migration script to support "list of values" and "file" repository columns/cells [fixes SCI-2165]

### PR DESCRIPTION
**Pull request by ZmagoD**
https://biosistemika.atlassian.net/browse/SCI-2165

### CODE REVIEW

**Comment by ZmagoD** [comment]
Smart annotations are handled in the team_import script.
Regarding S3_BUCKET probably we should use ENV['S3_BUCKET'] there may be @okriuchykhin has a comment on it.

**Comment by ZmagoD** [comment]
https://github.com/biosistemika/scinote-web/blob/master/app/services/team_importer.rb#L151

**Comment by Ducz0r** [comment]
Okay, let me be more precise: users can now reference individual repository items (this was previously **not possible**) using smart annotations. This results in a smart annotation that looks like this: `[#ILT/1~rep_item~1]`.

And these smart annotations for repository items are not updated when importing the data, they are left as it is, and this results in them pointing to either `nil` repository items, or possibly to items in other teams to which user doesn't have rights.

**Comment by ZmagoD** [comment]
Aha OK, I will fix also that.

**Comment by okriuchykhin** [comment]
S3_BUCKET is also ruby constant representing aws sdk s3 resource.

**Comment by ZmagoD** [comment]
@okriuchykhin so here we can just check if ENV['S3_BUCKET'] ?

File: app/services/team_importer.rb [hunk 1]
```diff
@@ -202,7 +190,7 @@ def update_smart_annotations(team)
   def update_annotation(text)
     return false if text.nil?
     updated = false
-    %w(prj exp tsk sam).each do |name|
+    %w(prj exp tsk).each do |name|
```
Comment (Ducz0r) [inline app/services/team_importer.rb]: You also need to include `rep_item` here; then the import should finally work as intended!

**Pull request merged**